src/style.css

```
@@ -1 +1,90 @@
-/* Styles go here */
+.body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  padding-right: 50px;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+}
```

make header as flex container and then `justify-content: space-between` + you will no need to use position absolute for your image
```suggestion
.header {
  padding-right: 50px;
  background-color: rgba(255, 255, 255, 1);
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
  font-family: Roboto, Arial, sans-serif;
  font-size: 12px;
  line-height: 14px;
  font-weight: 500;
}
```
```suggestion
.header {
  padding: 0 50px;
  background-color: rgba(255, 255, 255, 1);
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
  font-family: Roboto, Arial, sans-serif;
  font-size: 12px;
  line-height: 14px;
  font-weight: 500;
}
```